pipeline/load.py

```diff
@@ -0,0 +1,51 @@
+"""Loads the cleaned data into an AWS RDS database."""
+
+from os import environ
+
+from dotenv import load_dotenv
+import pandas as pd
+from sqlalchemy import create_engine, sql
+
+
+def get_database_connection():
+    """
+    Establishes a database connection to the database specified.
+    """
+    try:
+        engine = create_engine(
+            f"mssql+pymssql://{environ['DB_USER']}:{environ['DB_PASSWORD']}@{environ['DB_HOST']}/?charset=utf8")
+
+        return engine.connect()
+
+    except ConnectionError as error:
+        print(error)
+    pass
```

Comment:
maybe raise an error instead of passing ? since you'll get an error anyway when trying to do something with None